libs/gundeck-types/src/Gundeck/Types/Push/V2.hs

```diff
@@ -177,6 +183,9 @@ instance FromJSON ApsData where
 -----------------------------------------------------------------------------
 -- Priority
 
+-- | REFACTOR: do we ever use LowPriority?  to test, (a) remove the constructor and see what goes
+-- wrong; (b) log use of 'LowPriority' by clients in production and watch it a few days.  if it is
```

Comment:
Matthias added a comment (thanks!), but ideally I'd like to move faster on this and investigate the potential removal now instead of not-now.

Given that `native_priority` seems to be unused in client code, I think it's a sane approach to deprecate it in a separate PR (just reply with 400) and see whether staging breaks. I'll ask QA about this.

Reply:
i am planning to go through the `REFACTOR` comments soon, but for now i think this pull request is really ripe and shouldn't contain any more of it.  (not sure you are suggesting that, just saying...)

Reply:
Yeah, I wasn't suggesting doing it in this PR.

Reply:
I asked @red-avtovo and he pointed out that since notifications aren't very reliable at the moment, QA might not notice that they are even more broken. So, let's wait with removing LowPriority.